frontends/common/parser_options.h

```diff
@@ -43,6 +43,9 @@ class ParserOptions : public Util::Options {
     // annotation names that are to be ignored by the compiler
     std::set<cstring> disabledAnnotations;
 
+    // used to generate dump file names
+    mutable size_t dump_uid;
```

Comment:
```suggestion
    mutable size_t dump_uid = 0;
```

... and drop the init in the constructor.